src/model/ai/AlphaBetaSearchNode.java

```diff
@@ -38,22 +42,18 @@ public AlphaBetaSearchNode(Board board, int depthLevel, PieceColor currentPlayer
         return this.children;
     }
 
-    public Board getBoard() {
-        return this.board;
-    }
-
     public Move getMoveThatGotToThisState() {
         return this.moveThatGotToThisState;
     }
 
-    public boolean isLeaf() {
-        return (this.depthLevel == 0 || this.board.isEndState());
-    }
-
     public double getValue() {
         return this.value;
     }
 
+    public boolean isLeaf() {
+        return (this.depthLevel == 0 || this.board.isEndState());
+    }
+
     public void setValue(double value) {
         this.value = value;
     }
```

Comment:
I'm also auto-formatting everything, oh well, we'll just have to do with some meaningless sections in the diff. Shouldn't be a biggie.